integration-test-example.js

```diff
@@ -0,0 +1,25 @@
+// Integration test file with intentional issues for commenting
+function processUserData(users) {
+    // Missing input validation
+    let results = [];
+
+    // SQL injection vulnerability
+    const query = "SELECT * FROM users WHERE id = " + users[0].id;
```

Comment:
🔧 Let's use parameterized queries for security: 

```suggestion
const query = 'SELECT * FROM users WHERE id = ?';
const result = db.query(query, [users[0
```

.id]);]